src/main/scala/it/pagopa/interop/tenantprocess/api/impl/TenantApiServiceImpl.scala

```diff
@@ -387,6 +387,70 @@ final case class TenantApiServiceImpl(
     }
   }
 
+  override def addCertifiedAttribute(tenantId: String, seed: CertifiedTenantAttributeSeed)(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerTenant: ToEntityMarshaller[Tenant]
+  ): Route = authorize(ADMIN_ROLE) {
```

Comment:
```suggestion
  ): Route = authorize(ADMIN_ROLE, M2M_ROLE) {
```
@beetlecrunch right?

Reply:
Yes, I wasn't thinking about the m2m flow.